### ISSUE TITLE
[ID] Recent activities dashboard should reflect `Awarded` activities

### ISSUE THREAD

**Issue by mhuang-usdr** [usdigitalresponse/usdr-gost#555]
Recent activities dashboard should reflect `Awarded` activities.

Right now the line shows up, but isn't a full sentence and doesn't have an icon.

For example, the image below should say:

>  USDR was **awarded** Private Sector Led Growth, Resilience and Response Activity

<img width="814" alt="image" src="https://user-images.githubusercontent.com/91221265/199086788-038ebe78-cdde-4fde-b63e-ada10a9faa6c.png">

open Q - what icon should be used?

**Comment by mhuang-usdr** [comment]
@cajinajr @rachel699 - do yuo have thoughts on what icons could be used for this?

**Comment by rachel699** [comment]
Hi Mindy: How about one of these icons for Award? 
<img width="32" alt="healthicons_award-ribbon-outline" src="https://user-images.githubusercontent.com/60455000/202817732-e22cfe0e-ecf1-489f-8708-b862c5e437e5.png">
<img width="32" alt="uil_award" src="https://user-images.githubusercontent.com/60455000/202817733-839a0cd9-4b3e-4ec5-a296-9f8827fe7ab0.png">
<img width="32" alt="mdi_award" src="https://user-images.githubusercontent.com/60455000/202817735-e928bf0a-2419-472e-9da0-567efb980252.png">

**Comment by mhuang-usdr** [comment]
@rachel699 I like the middle one! Is there an icon library you got that from?

**Comment by rachel699** [comment]
yes, I like the 2nd one as well. It's from a Figma plugin called Iconify. 
Sorry for delayed reply. I made a few versions of the icon here.

**Comment by rachel699** [comment]
![Vector-1](https://user-images.githubusercontent.com/60455000/202819097-d24eaf0d-1958-41f9-8905-f8173eaa4a4a.png)
![Vector](https://user-images.githubusercontent.com/60455000/202819111-9f74a67f-7f46-4155-8eec-ea533cef710f.png)
![Vector-2](https://user-images.githubusercontent.com/60455000/202819118-38bd7162-0737-43cb-afbd-6aa80b21b04c.png)
![Frame 1](https://user-images.githubusercontent.com/60455000/202819124-e76beb49-dfe8-4837-87b5-20e5b4428f1c.png)
![Frame 1](https://user-images.githubusercontent.com/60455000/202819130-ad9623bd-c535-49cc-aee3-a87b1cc61cdc.png)

**Comment by rachel699** [comment]
I can send the SVG file to the engineers, I think it'll work.

**Comment by rachel699** [comment]
Here are the SVG file of the above icon versions.
![Vector-1](https://user-images.githubusercontent.com/60455000/202819479-28a59cdf-609e-4b86-9d31-fd2d7a5fa5f1.svg)
![Vector](https://user-images.githubusercontent.com/60455000/202819491-5087c898-fb54-44b9-ad7a-6939f0446043.svg)
![Vector-2](https://user-images.githubusercontent.com/60455000/202819506-686bb4f7-f2f5-4297-8707-d3603ee46698.svg)
![Frame 2](https://user-images.githubusercontent.com/60455000/202819531-e42181b5-1588-4079-8c4c-532fb5414bdc.svg)
![Frame 1](https://user-images.githubusercontent.com/60455000/202819537-4209667a-f21a-4ae5-b678-99b49792eb88.svg)

**Comment by rachel699** [comment]
I think the above SVG files can be downloaded and used by engineers. Please let me know if there is any problem.

**Comment by mhuang-usdr** [comment]
Thank you!!

**Comment by cajinajr** [comment]
I like the middle one too. I know we’re using Bootstrap for the tools so
just in case it’s easier for the devs to implement, I don’t think it would
be a bad idea to consider Bootstrap’s icons for an award:

https://icons.getbootstrap.com/icons/award/

https://icons.getbootstrap.com/icons/award-fill/

**Comment by rachel699** [comment]
Or simply like this to be similar with the other icons in the current list. 
![Frame 3](https://user-images.githubusercontent.com/60455000/202820422-1fc45375-8104-486a-bca0-cdd1ac35a985.svg)

**Comment by cajinajr** [comment]
![Recent Activity - Icons - Awarded](https://user-images.githubusercontent.com/16214088/206297752-2c2b713b-828d-4fb1-bd43-faefac45a6d5.jpg)

Updated to reflect suggestions. The only issue I see now is that the gold color does not pass WCAG accessibility for text and graphics color contrast. I played around with a darker gold but it basically has to be muddy mustard color in order to pass. Maybe we could use the same green that's being used for interested?

**Comment by mhuang-usdr** [comment]
@cajinajr I'm okay with muddy mustard or the green, whichever you think looks best!

**Comment by TylerHendrickson** [comment]
Very similar to #622 ; consider working on these close together / in sequence
